### PR TITLE
Code Refactoring and Cleanup

### PR DESCRIPTION
**Pull request by dejv67**
This pull request includes a comprehensive refactor of the codebase to improve readability, maintainability, and performance. Unused or redundant code has been removed, naming conventions standardized, and overall code structure simplified without altering functionality.

### CODE REVIEW

File: NNPTPZ1/Mathematics/ComplexNumber.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        public ComplexNumber Add(ComplexNumber b)
+        {
+            ComplexNumber a = this;
+            return new ComplexNumber()
+            {
+                RealPart = a.RealPart + b.RealPart,
+                ImaginaryPart = a.ImaginaryPart + b.ImaginaryPart
+            };
+        }
+        public double GetAngleInDegrees()
```
Comment (rodi0878) [inline NNPTPZ1/Mathematics/ComplexNumber.cs]: really?

File: NNPTPZ1/Mathematics/ComplexNumber.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    {
+        public readonly static ComplexNumber Zero = new ComplexNumber()
+        {
+            RealPart = 0,
+            ImaginaryPart = 0
+        };
+        public double RealPart { get; set; }
+        public double ImaginaryPart { get; set; }
+
+        public override bool Equals(object obj)
```
Comment (rodi0878) [inline NNPTPZ1/Mathematics/ComplexNumber.cs:17]: fix ordering of members in class, group common things together

**Review by rodi0878 — CHANGES_REQUESTED** [review]
Please review mentioned problematic parts of code

**Comment by dejv67** [comment]
requested changes fixed

**Comment by rodi0878** [comment]
ok, accepted

**Pull request closed without merging**